packages/shared/src/lib/log.ts

```diff
@@ -281,6 +284,7 @@ export enum TargetId {
   Ads = 'ads',
   MyProfile = 'my profile',
   PlusBadge = 'plus badge',
+  Onboarding = 'onboarding',
```

Comment:
I think we have this already, but maybe not id?

Reply:
We have it on `Origin`, should I use this instead?